index.bs

```diff
@@ -2536,6 +2536,24 @@ offers guidance that should be considered in the development of new features, no
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3 id="consider-dedicatedworker-support">Where possible APIs should be made available in DedicatedWorker</h3>
+
+When exposing a feature, please consider whether it makes sense to expose the feature
+to DedicatedWorker as well.
+A DedicatedWorker is a non specialized worker type that allows developers to run code
+off the main thread.
```

Comment:
I think we can just reference back to DedicatedWorker and not explain this ourselves.

Reply:
But then devs will have to open multiple links, maybe do both?